QUEST_JOBSTEP.tsv

```diff
@@ -1328,13 +1328,13 @@ QUEST_JOBSTEP_20150317_001327	The Ranger Master is looking for a book and he say
 QUEST_JOBSTEP_20150317_001328	Give book to the Ranger Master
 QUEST_JOBSTEP_20150317_001329	Found the book the Ranger Master is looking for. Give the book to the Ranger Master.
 QUEST_JOBSTEP_20150317_001330	Get Lydia Schaffen's book from Vubbe Fighter
-QUEST_JOBSTEP_20150317_001331	This isn't the only book the Ranger Master is looking for. Talk to the Ranger Master again.
-QUEST_JOBSTEP_20150317_001332	The Hunter Master has the book the Ranger Master wants. Prepare 5000 silvers to get the book from the Hunter Master.
-QUEST_JOBSTEP_20150317_001333	The Archer Master doesn't express it much but he seems happy to have found the book. Talk to him about the next book to find.
-QUEST_JOBSTEP_20150317_001334	The Ranger Master told you that you may be able to trade with the Quarrel Shooter Master with the Merrogg Leather for the book. Obtain the Merrogg Leather from Fallen Worry Forest.
-QUEST_JOBSTEP_20150317_001335	The Ranger Master looks very happy to have found the book. Talk to him to find the next book.
-QUEST_JOBSTEP_20150317_001336	Collect rocks from the area the Ranger Master mentioned
-QUEST_JOBSTEP_20150317_001337	Collected all the books for the Ranger Master. Talk to the Ranger Master to be acknowledged as a Ranger.
+QUEST_JOBSTEP_20150317_001331	This isn't the only book Ranger Master is looking for. Talk to the Ranger Master again.
```

Comment:
This isn't the only book Ranger Master is looking for -> This isn't the only book that the Ranger Master is looking for